modules/nw-dpu-configuring-operator.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/configuring-dpu-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-dpu-configuring-operator_{context}"]
+= Configuring the DPU Operator 
+
+Create a `DpuOperatorConfig` custom resource (CR) on both the host-side cluster and on each of the dpu-side clusters. The DPU operator in each cluster is activated after this CR is created.
+
+. Create a file named `dpu-operator-host-config.yaml` using the following YAML:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: DpuOperatorConfig
+metadata:
+ name: dpu-operator-config <1>
+spec:
+ mode: host <2>
+----
++
+<1> The name of the custom resource must be `dpu-operator-config`.
+<2> Set the value to `host` on the host-side cluster. On each DPU-side cluster, which runs a single MicroShift cluster per DPU, set the value to `dpu`.
+
+. Create the resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f dpu-operator-host-config.yaml
+----
+
+. All nodes in the host-side cluster that have an attached DPU running MicroShift should be labeled with `dpu=true`. You can apply this label by running the following command:
```

Comment:
Add after first sentence: The sole node in the microshift cluster should also be labeled with `dpu=true`.